InternetHospital.DataAccess/AppContextConfiguration/RoleConfiguration.cs

```diff
@@ -11,26 +13,33 @@ public class RoleConfiguration
     {
         public static async Task InitializeAsync(RoleManager<IdentityRole<int>> roleManager)
         {
-            const string PATIENT = "Patient";
-            const string DOCTOR = "Doctor";
-            const string MODERATOR = "Moderator";
-            const string ADMIN = "Admin";
-         
-            if (await roleManager.FindByNameAsync(PATIENT) == null)
+            try
             {
-                await roleManager.CreateAsync(new IdentityRole<int>(PATIENT));
-            }
-            if (await roleManager.FindByNameAsync(DOCTOR) == null)
-            {
-                await roleManager.CreateAsync(new IdentityRole<int>(DOCTOR));
+                string json = File.ReadAllText(@"..\InternetHospital.DataAccess\AppContextConfiguration\RoleConfigurationJSON.json");
+
+                var jsonRoles = JArray.Parse(json);
+
+                string role = null;
+
+                foreach (dynamic item in jsonRoles)
+                {
+                    //It's necessary because i had an exception in runtime execution
+                    role = item.Name;
+                    if (await roleManager.FindByNameAsync(role) == null)
+                    {
+                        await roleManager.CreateAsync(new IdentityRole<int>(role.ToString()));
```

Comment:
role is already a string there is no need in calling ToString().

Reply:
Done